redirects/MANUAL.md

```diff
@@ -0,0 +1,59 @@
+# Redirect rules manual
+
+## What are redirect rules?
+
+Redirect rules allow redirecting users to a new documentation page when they land on an old page
+that was either renamed or moved elsewhere. They are specified in `.txt` files located in the
+`redirects/` directory at the root of the documentation. Each line of these files specify a single
+rule that applies to a single documentation page.
+
+## How do redirect rules work?
+
+For each redirect rule, the 'redirects' Sphinx extension creates a blank HTML file at the location
```

Comment:
I think `redirects` doesn't render as intended
```suggestion
For each redirect rule, the `redirects` Sphinx extension creates a blank HTML file at the location
```
![image](https://user-images.githubusercontent.com/46810843/175031159-159d1f6a-0cd9-47ea-8885-59c25b16ad7e.png)

Reply:
It was but it's maybe not the best way to mention it. It's a name so I'll simply remove the quotes.